### ISSUE TITLE
Stop button is not working

### ISSUE THREAD

**Issue by vbarbarosh** [foss-project/green-recorder#121]
After start recording the only way to stop is to close application. Clicking on **stop** button has no effect.

```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.1 LTS
Release:	18.04
Codename:	bionic
```
Version of **green-recorder** is 3.2. I tried to open it from terminal. This is what I get by clicking on **stop** button:

```
TypeError: stoprecording() takes exactly 1 argument (0 given)
Traceback (most recent call last):
  File "/usr/bin/green-recorder", line 484, in stoprecordingclicked
    stoprecording()
```

**Comment by mhsabbagh** [comment]
This sounds like an issue from the code indeed and I'll fix it soon. But it's not the only way to stop recording; You can stop recording using the system tray icon.

**Comment by vbarbarosh** [comment]
I fixed it by changing

```
def stoprecording(_):
```

to
```
def stoprecording():
```

Concerning your recommendation of using system tray icon - I didn't have it. So, closing **green-recorder** was the only option for me.

https://i.ibb.co/2P17HBv/Screenshot-from-2019-02-09-14-57-38.png

Btw, thanks for the app.

**Comment by mhsabbagh** [comment]
Fixed in 3.2.1
